frontend/models/product-list/index.ts

```diff
@@ -134,6 +121,35 @@ function getDeviceImage(deviceWiki: DeviceWiki): ProductListImage | null {
         };
 }
 
+async function fillMissingImagesFromApi(
+   productListChildren: ProductListChild[]
+): Promise<ProductListChild[]> {
+   const childrenWithoutImages = productListChildren.filter(
+      (child) => child.image == null && child.deviceTitle
+   );
+   if (childrenWithoutImages.length === 0) {
+      return productListChildren;
+   }
+   const deviceTitlesWithoutImages = childrenWithoutImages.map(
+      (child) => child.deviceTitle
+   ) as string[]; // cast is safe cause we filter nulls above,
+                  // typescript just doesn't understand
+   const imagesResponse = await fetchMultipleDeviceImages(
+      deviceTitlesWithoutImages,
+      'standard'
+   );
+   childrenWithoutImages.forEach((child) => {
+      const imageFromDevice = imagesResponse.images[child.deviceTitle as string];
+      if (imageFromDevice != null) {
+         child.image = {
+            url: imageFromDevice,
+            alternativeText: null,
```

Comment:
Might as well make this the device title.
```suggestion
            alternativeText: child.deviceTitle,
```